### DISCUSSION TITLE
hideable toc

### DISCUSSION THREAD

**Discussion by dirslashls** [facebook/docusaurus#7809] [Ideas]
I like the ability to let the end user hide the sidebar. I am looking for a similar configuration which allows end user to expand/collapse the TOC section. Also, is it possible to configure such that the TOC shows as a dropdown menu on the top (like it does when the browser width is smaller) for all device sizes?

**Comment by slorber** [comment]
> I am looking for a similar configuration which allows end user to expand/collapse the TOC section.

Please show us a mockup of this UX or an actual doc site implementing a similar pattern. We should have a clear idea on what it should look like and behave before even thinking implementing this.

> Also, is it possible to configure such that the TOC shows as a dropdown menu on the top (like it does when the browser width is smaller) for all device sizes?

You can use swizzle to force the usage of a collapsible menu on all viewport sizes: `@theme/DocItem/Layout`

We can't provide config options for every UI possibility, and will only do if a good number of users are sharing the same use-case (ie we don't implement a feature just for 1 user)

Please show us your UX implemented in userland first, and we'll consider moving it officially in core if it makes sense
